packages/plugin-audio-button-response/src/index.ts

```diff
@@ -282,6 +293,14 @@ class AudioButtonResponsePlugin implements JsPsychPlugin<Info> {
       }
     }
 
+    function enable_buttons() {
+      if (trial.enable_button_after !== null) {
```

Comment:
If `enable_button_after` is set to `0`, then the timeout will still trigger and this may create unpredictable behavior that depends on the JS event loop. I think it is better to check if it is `null` or `0` (or simply make the default `0`)